### ISSUE TITLE
Something kills vampire hunters

### ISSUE THREAD

**Issue by yuno44907** [TeamLapen/Vampirism#545]
Someone kept telling me make a feature request. You completely need something to kill Vampire Hunters. because Vampires weak to sunlight, silver, fire, anti undead damage. And Vampire Hunters strong to undeads vampires werewolves.

What we have? Water damage water deals damage to enderman; vulcanite mod adds items that strong against fire resist mobs and players; daylight; undead i dont know if is it possible to add undeads that immune to day light Zombie Players mod adds undeads that immune to daylight but this zombie players crashes with Requiem mod; normal damage; fall damage; void damage; passive mobs; neutral mobs; aggressive mobs; players.

So what i am saying is by using tags already exist in game we can add new classes. Dragons weak to vampires, strong to everyone that is not a dragon hunter, specially strong against werewolves and minotaurs. Hardest thing to level up, nearly no level cap. Can do everything ice and fire dragons does. Can call thunder to the target if underground thunder hits surface. Consumes too much hunger. Has a secondary bar that bar fills by destroying items like eating them but you dont eat instead destroy like throwing to garbage. Everyone know where is the dragon while it is in dragon form. Dragons are normally human magicians they become dragon with transformation spell because this is how half human half dragons work. When dragon dies it drop very valuable items based on its EXP and level. Dragon hunter element is strong against dragons lizards and magicians and magical things and large wild beasts.

Spider Queen was a mod. It will be weak to fire. It counts as a spider. It can tame spiders keep them in inventory, takes 1 slot each one. Lay eggs. If it takes damage all spiders near will defend it. It can generate strings and can craft webs with strings. Sticky things doesnt slow it including soulsand. Everything that is not spider or passive will attack to spider eggs. It can leave spiders for defend near to nest. Defending spiders will attack and return to place where they were waiting after they kill the target otherwise they will never return. If something hits an egg they go after that enemy.

Arcane Skeleton can shoot fire balls. Can burn. Can call lightning. Entities that effected by lightning he called will be tamed. Can tame fire resist mobs. Weak to ice. Gets more slowness debuff than other mobs even soulsand slows it terribly. Takes damage from water. Weak to vulcanite mod.

Seaman at start it cant breath outside of water. Has infinite water breath. Water streams doesnt move him. Can gain swift swim. At start walks slow on land 0,9. Cant sprint. Can spawn water source blocks.. Later levels can create snow and turn water to ice. Can generate pumpkin for building snow golems. Can use ice and fire mod ice attacks. Doesnt have a weakness because it is weak, it is boring.

Minotaurs walk slower than players. Has smash skill that doubles your next damage and stays until you deal damage to an entity. This skill has high cooldown. Can move with same speed later levels. Very strong and hard to kill. Drop meat when die based on level. May try to add Mowzies mob style skills and block breaking dmg dealing area abilities. Can be resistant to fire and poison and ice and thunder. Charge skill gains move speed buff and damage buff until it deals damage after that buff ends. Smash deals more damage.

Turtle, Tiger... Whatever, i have to make vampirism, howling moon, dissolution, zombie players, corpse, hardcore revival mods work together. Dissolution ghosts get stuck in corpse at first transformation kill command 2 times fix problem. If ghost posses an Zombie player and die as zombie player game crash any other mobs doesnt crash anything.

Guys Zombie Players mod is crashing if a player from Requiem or Dissolution mod in ghost or Remnant state takes control of the Zombie Player mob and dies game crash. If zombie player is coded as an entity and game trying to spawn another Zombie player that can be reason. Do you know any mods that spawn mob when player dies like becoming a zombie when die; years ago there was skeleton chest when you die a skeleton with your inventory spawns and it takes you as a group when you hit it. I am trying to edit code but there is too much code; i cant find where i have to edit.

**Comment by PendragonII** [comment]
First thing to know:

 - Vampires are generalized roleplay characters: Turning to bats to fly, passive speed with each level increase, ignored by fellow _vampires_ and _undead zombies_, teleportation, strength from rage. Indeed, Vampires are generalized and well fitted for many roleplayers.
 - Hunters are specialized roleplay characters: Hunters specialize in hunting undead such as Vampires. Holy Water, Crossbows, Garlic, all these things are tailored to hunting Vampires. This roleplay style is combat focused.

**Vampires being generalized, do have combat capabilties: here are some ingame screeshots:**
![2019-10-13_10 57 46](https://user-images.githubusercontent.com/43933998/66717601-ceb88a80-eda8-11e9-967b-a990c59cbb1c.png)

 - Vampire Rage: Temporary speed and strength boost. Passive recharge.
 - Advanced Biter: Increase biting damage.
 - Finisher: Instantly kill any _non-vampire_ below 25% max health using a _Vampire Sword_

![2019-10-13_10 57 55](https://user-images.githubusercontent.com/43933998/66717654-5ef6cf80-eda9-11e9-874f-8087403ac48b.png)

 - Dark Blood Projectile: Magic, send a lethal projectile that conducts area damage. Short passive recharge.
 - Blood Charge: Recharge your _Vampire Sword_ on the spot (mist battle, prebattle, anywhere) using your hunger points, removing the Blood Pedestal liability.
 - **Freeze**: Freeze enemy combatants (or any entities) where they are on-the-spot. Allowing Vampires (you) to hit them without the enemy being able to flee for a short period of time. Passive Recharge.

Hunters specialize in hunting Vampires, sacrificing the generalization benefits toward focusing on this roleplay style.

> You completely need something to kill Vampire Hunters.

Vampires have combat skills, read above.

**TL;DR:** It would break the balance between Vampires and Hunters if Vampires were both generalized _and_ combat dedicate toward Hunters.

--

> vulcanite mod adds items [...]

>  Zombie Players mod adds [...]

> [...]  with Requiem mod;

**Etcetera**

This mod is dedicated toward Vampires and Vampire Hunters. If there is a conflict between Vampirism and another mod please open a dedicate bug report to help the mod developers.

**Comment by yuno44907** [comment]
You told me to write a feature request and now you write TL DR; by the way 3 years have been passed you didnt add anything there is only vampires and hunters. Everyday you delete and block multiple people; i hope god doesnt behave like you.

**Comment by maxanier** [comment]
[20:34] yuno44907: Dude this is how this mod is
[20:34] yuno44907: And this person will not code this

You explained it yourself.

**Comment by yuno44907** [comment]
By the way i noticed Dissolution mod has very hard to understand and follow coding. They write action1b and you try to understand what is action1b you go to core mod library and stuff. You know this is evil coding that really coded it scrambled it. I have seen other game files like Darkest Dungeon or Battletech all was very simple. Maybe java is harder to understand than c+ but everyone say java easier to learn. I have to read more code for understanding what is going on i didnt read vampirism code yet.
